scheduler/reconcile.go

```diff
@@ -190,8 +189,8 @@ func (a *allocReconciler) Compute() *reconcileResults {
 	// Create the allocation matrix
 	m := newAllocMatrix(a.job, a.existingAllocs)
 
-	// Handle stopping unneeded deployments
-	a.cancelDeployments()
+	// Stop unneeded deployments
+	a.cancelUnneededDeployments()
```

Comment:
If changing the name didn't obviate the comment, then it's the wrong name 😁 

```suggestion
	a.cancelUnneededDeployments()
```